test/unit/spec/signaling/v2/peerconnection.js

```diff
@@ -960,7 +960,8 @@ describe('PeerConnectionV2', () => {
       async function setup() {
         let tracks;
         if (chromeScreenTrack) {
-          tracks = [{ id: 'foo', kind: 'video', label: 'screen:123' }];
+          const getSettings = () => { return { width: 1280, height: 720 }; };
+          tracks = [{ id: 'foo', kind: 'video', label: 'screen:123', getSettings }];
```

Comment:
Is it possible to test the combinations that don't result in 3 encodings as well? IIUC only 1280x720 is tested.

Reply:
yes, I am adding more unittests targetting various encodings.

Reply:
I have added few tests to check various resolutions.